### PR TITLE
Pract.05: Add 02 | Bozhidar Tomov

### PR DESCRIPTION
**Pull request by Bozhidar-Tomov**
Bozhidar Tomov
SI 1 0MI0600171

### CODE REVIEW

**Comment by Justsvetoslavov** [comment]
Can you rename task due to added tasks in https://github.com/Justsvetoslavov/Object-oriented_programming_FMI/blob/master/Seminars/Sem.05/Pract.%2005/README.md?

**Comment by Bozhidar-Tomov** [comment]
Done
before: ~~Pract.05: Add 01 | Bozhidar Tomov~~
new: **Pract.05: Add 02 | Bozhidar Tomov**

File: Seminars/Sem.05/Pract. 05/Task01/Task02.h [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    double getReal() const;
+    double getImaginary() const;
+
+    bool isEqual(const ComplexNumber &, const ComplexNumber &) const;
+    void conjugate();
+    ComplexNumber getConjugated() const;
+    void print() const;
+};
+
+ComplexNumber add(const ComplexNumber &, const ComplexNumber &);
```
Comment (Justsvetoslavov) [inline Seminars/Sem.05/Pract. 05/Task01/Task02.h:25]: Shouldn't these 3 be friend functions ? If yes, why?

File: Seminars/Sem.05/Pract. 05/Task01/Task02.cpp [hunk 2]
```diff
@@ -0,0 +1,66 @@
+#include <iostream>
+#include "Task02.h"
+
+const double EPSILON = 0.0001;
+
+double absolute(double x)
+{
+    return x < 0 ? -x : x;
```
Comment (Justsvetoslavov) [inline Seminars/Sem.05/Pract. 05/Task01/Task02.cpp:8]: Absolute in doubles, how is it determined?

**Review by Justsvetoslavov — CHANGES_REQUESTED** [review]

**Pull request closed without merging**